core/html_api.php

```diff
@@ -765,10 +746,21 @@ function html_footer() {
 }
 
 /**
- * (14) End the <body> section
+ * (12) End the <body> section
  * @return void
  */
 function html_body_end() {
+	global $g_scripts_included;
+
+	echo "\t" . '<script type="text/javascript" src="' . helper_mantis_url( 'javascript_config.php' ) . '"></script>' . "\n";
+	echo "\t" . '<script type="text/javascript" src="' . helper_mantis_url( 'javascript_translations.php' ) . '"></script>' . "\n";
+	html_javascript_link( 'jquery-1.11.3.min.js' );
+	html_javascript_link( 'jquery-ui-1.11.4.min.js' );
+	html_javascript_link( 'common.js' );
+	foreach ( $g_scripts_included as $t_script_path ) {
+		html_javascript_link( $t_script_path );
+	}
+
 	event_signal( 'EVENT_LAYOUT_BODY_END' );
```

Comment:
Should the body end event proceed the javascript, so the javascript is right before the closing body tag?